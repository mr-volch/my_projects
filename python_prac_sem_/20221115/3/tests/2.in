from time import time
t0 = time()
alp = Alpha(c=10, z=2, a=42)
alp.e = 123
alp.b = 123
alp.d = 123
alp.e = 123
alp.f = 123
alp.g = 123
alp.h = 123
alp.i = 123
alp.j = 123
alp.k = 123
alp.m = 123
print(alp)
t01 = time() - t0
t1 = time()
alq = AlphaQ(c=10, z=2, a=42)
alq.e = 123
alq.b = 123
alq.d = 123
alq.e = 123
alq.f = 123
alq.g = 123
alq.h = 123
alq.i = 123
alq.j = 123
alq.k = 123
alq.m = 123
print(alq)
t11 = time() - t1
print(f"{t11<t01=}")
