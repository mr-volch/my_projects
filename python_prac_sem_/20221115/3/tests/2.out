a: 42, b: 123, c: 10, d: 123, e: 123, f: 123, g: 123, h: 123, i: 123, j: 123, k: 123, m: 123, z: 2
a: 42, b: 123, c: 10, d: 123, e: 123, f: 123, g: 123, h: 123, i: 123, j: 123, k: 123, m: 123, z: 2
t11<t01=True
